Tools/PowershellModule/doc/new-winget-rest-source-azure.md

```diff
@@ -1,122 +1,118 @@
-# Windows Package Manager REST source
+# Create a REST source Windows Package Manager
```

Comment:
Huh? This string flipped?

shouldn't this be "Create a Windows Package Manager Rest source"?